### DISCUSSION TITLE
Doctr throws an error when used for detection_predictor

### DISCUSSION THREAD

**Discussion by nehashinde-canopy** [mindee/doctr#1491] [Q&A]
This is my code:
```
from doctr.models import detection_predictor
predictor = detection_predictor('db_resnet50', pretrained=True, assume_straight_pages=False, preserve_aspect_ratio=True)
result = predictor('/path/to/image/91ef4b7f-ktp-74_jpg.rf.225742a25068907cf3be6b52593208ccy.jpg')
```
AttributeError: 'str' object has no attribute 'ndim'
I have installed doctr pytorch version using: pip install "python-doctr[torch]"
I have a linux system.
Kindly help.

**Comment by felixdittrich92** [comment]
```
from doctr.models import detection_predictor
from doctr.io import DocumentFile

predictor = detection_predictor('db_resnet50', pretrained=True, assume_straight_pages=False, preserve_aspect_ratio=True)

doc = DocumentFile.from_images(["/path/to/image/91ef4b7f-ktp-74_jpg.rf.225742a25068907cf3be6b52593208ccy.jpg"])  # returns np.ndarray
result = predictor(doc)
```

**Reply by nehashinde-canopy** [reply]
Thanks! this works but it outputs an array:
[{'words': array([[[    0.79677,     0.51756],
        [    0.87578,      0.5618],
        [    0.84093,     0.62404],
        [    0.76191,      0.5798]],

How to interpret this result? I need rotation degrees of tilted images. I am using detection_predictor for rotation detection of the input image.

**Comment by whoami02** [comment]
Those are point coordinates, multiply them by image size. @nehashinde-canopy 
```
w,h = img.size
for j in coords:
      j = [j[0]*h, j[1]*w]
```
Once you get the bounding box, get the angle by comparing it with a line between (0,0) and (w,0)
